changelogs/fragments/1-postgres.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- postgresql - added new parameters ssl_cert and ssl_key for ssl connection (https://github.com/ansible-collections/community.postgresql/issues/424).
```

```suggestion
- postgresql - added new parameters ``ssl_cert`` and ``ssl_key`` for ssl connection (https://github.com/ansible-collections/community.postgresql/issues/424).
```
to highlight them in rst a bit